src/algorithms/calorimetry/NeutronReconstructionConfig.h

```diff
@@ -0,0 +1,12 @@
+// Subject to the terms in the LICENSE file found in the top-level directory.
+
+#pragma once
+
+namespace eicrecon {
+
+struct NeutronReconstructionConfig {
+
+
+};
```

Comment:
Should there be parameters here? If not, I'd suggest using `WithPodConfig<NoConfig>` and removing this file.

Reply:
Done as requested